### PR TITLE
Ubuntu 20.04

### PR DESCRIPTION
**Pull request by Mustard**
Issue https://github.com/mail-in-a-box/mailinabox/issues/1751

### CODE REVIEW

File: setup/bootstrap.sh [hunk 1]
```diff
@@ -32,9 +43,8 @@ if [ -z "$TAG" ]; then
 		echo "to Ubuntu 18.04."
 		echo ""
 		TAG=v0.30
-
 	else
-		echo "This script must be run on a system running Ubuntu 18.04 or Ubuntu 14.04."
+		echo "This script must be run on a system running Ubuntu LTS versions 20.04, 18.04, or 14.04"
```
Comment (octoquad) [inline setup/bootstrap.sh:47]: Ubuntu 14.04 is [end of life](https://wiki.ubuntu.com/Releases) and is only available via Extended Security Maintenance (ESM).

**Comment by alibitek** [comment]
@Mustard 
Did you test this PR in a VPS with Ubuntu 20.04 LTS ? 
Everything worked fine?
Any issues you encountered? 
What does /admin page say? Everything green?

**Comment by tutosfaciles48** [comment]
Hello, any news on this PR ?

**Comment by JoshData** [comment]
Thanks for getting this started. I've moved/updated the changes to #2083.

**Pull request closed without merging**